### PR TITLE
New package: Sentinel.SentinelCLI version 0.3.2

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135290)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 0.3.2
+PackageLocale: en-US
+Publisher: Sentinel
+PublisherUrl: https://sentinel.co
+PublisherSupportUrl: https://github.com/sentinel-official/cli-client/issues
+PackageName: Sentinel CLI
+PackageUrl: https://github.com/sentinel-official/cli-client
+License: APACHE-2.0
+LicenseUrl: https://github.com/sentinel-official/cli-client/blob/master/LICENSE
+ShortDescription: The official Sentinel CLI client
```
Comment (Trenly) [inline manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.locale.en-US.yaml:14]: Please provide a better description of what this CLI is, does, or is used for

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Sentinel.SentinelCLI
+PackageVersion: 0.3.2
+InstallerType: burn
```
Comment (stephengillie) [inline manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.installer.yaml:6]:
```suggestion
InstallerType: portable
```

Comment (stephengillie) [reply manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.installer.yaml:6]: With another installer type, Validation downloads the EXE, runs it, then deletes it (thinking it was the installer) - causing the ` Validation-No-Executables` label.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**